### PR TITLE
Updating Underscore to resolve SNYK-JS-UNDERSCORE-1080984

### PR DESCRIPTION
**Pull request by jessehouwing**
This change is [Reviewable](https://reviewable.io/reviews/microsoft/typed-rest-client/258)

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,6 +42,6 @@
   "dependencies": {
```
Comment (anatolybolshakov) [inline package.json:42]: Could you please bump package version (patch number) also?

**Comment by anatolybolshakov** [comment]
Hi @jessehouwing thanks for bringing it up! Let us test these change also.

**Comment by PierreFritsch** [comment]
@anatolybolshakov can you communicate an ETA for the `underscore` version upgrade?

**Review by DaniilShmelev — APPROVED** [review]
Tested these changes manually, found no issues. LGTM, thanks!

**Review by anatolybolshakov — APPROVED** [review]

**Pull request merged**